src/Symfony/Component/Notifier/Bridge/Telegram/README.md

```diff
@@ -14,6 +14,18 @@ where:
  - `TOKEN` is your Telegram token
  - `CHAT_ID` is your Telegram chat id
 
+Interacting with a local API server instead of the official Telegram API
+------------------------------------------------------------------------
```

Comment:
```suggestion
Interacting with a self-hosted API server instead of the official Telegram API
------------------------------------------------------------------------------
```

It looks like this change would allow to connect to any Telegram server, not only a local server.

Reply:
Nevermind, I just saw that Telegram describes it as “local”: https://core.telegram.org/bots/api#using-a-local-bot-api-server

See: 
- #57766